### DISCUSSION TITLE
open a "gitignored" file?

### DISCUSSION THREAD

**Discussion by well1791** [helix-editor/helix#6066] [Q&A] [answered]
Hi everyone, I wonder if there is a way to open an ignored file within the same directory either by using the search function `SPC F` or by autocomplete on the cli: `:o ignored-file.ext`

**Comment by the-mikedavis** [comment, marked as answer]
Ignored files don't show up in the file picker (`<space>f`) and aren't auto-completed in `:open` but you can open them my proving the full path to the file with `:open path/to/file`. You can also change the settings for which files are ignored for the file picker https://docs.helix-editor.com/master/configuration.html#editorfile-picker-section

**Comment by ScreenTurnOn** [comment]
The file explorer is an option that allows for fuzzy finding.

`<space> + e` opens in workspace root
`<space> + E` opens in current buffer's directory
